src/web_ui/__init__.py

```diff
@@ -159,6 +161,40 @@ def wifi_ap_status():
     )
 
 
+@app.route("/auto_config", methods=['GET', 'POST'])
+def auto_config():
+    if request.method == 'POST':
+        width = request.form["Width"]
```

Comment:
I think we can just use lowercase names for these

Reply:
okay